docs/tr/extend/testing.md

```diff
@@ -1 +1,3 @@
-# Testing
\ No newline at end of file
+# Testing
```

Comment:
Perhaps these page titles should be translated too, even if they're not populated yet?

Reply:
I do not know the Turkish of some titles. I will search and update.

Reply:
Thanks!

Reply:
@askvortsov1 I updated the titles.